UACloudLibraryServer/UA-CloudLibrary.csproj

```diff
@@ -19,6 +19,7 @@
         <PackageReference Include="Google.Cloud.Storage.V1" Version="3.7.0" />
         <PackageReference Include="Google.Cloud.AspNetCore.DataProtection.Storage" Version="1.0.0-alpha03" />
 	<PackageReference Include="GraphQL" Version="4.7.1" />
+	<PackageReference Include="GraphQL.EntityFramework" Version="15.2.0" />

```

Comment:
As discussed, please try to remove, if possible.

Reply:
GraphQL.EntityFramework is based on the GraphQL Nuget-Package and adds the EntityFramework-Layer ontop of GraphQL
See: https://www.nuget.org/packages/GraphQL.EntityFramework/#dependencies-tab 
And https://github.com/SimonCropp/GraphQL.EntityFramework

It is the most recent version of Database-Layers for the GraphQL-Engine: https://graphql-dotnet.github.io/docs/getting-started/databases